helpers/yum_repositories.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python
+# Copyright (c) 2013-2015 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 3 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE about this file by physical or electronic mail,
+# you may find current contact information at www.suse.com
+
+import yum
+import json
+
+yb = yum.YumBase()
```

Comment:
I am no yum expert but I inspected several times the repositories while installing packages on the same machine and didn't had any issues. Since this is only reading anyway I think we are fine.